packages/components/src/button/README.md

```diff
@@ -124,6 +124,7 @@ The presence of a `href` prop determines whether an `anchor` element is rendered
 
 Name | Type | Default | Description
 --- | --- | --- | ---
+`disabled` | `bool` | `false` | Whether the button is disabled.
```

Comment:
Hi @rmccue, thank you for your contribution 👍 
Our code does not explicitly handle the disable property it ends up being supported because the props not documented here are passed as attributes to the "a" or "button" elements. Any HTML attribute supported by these elements will work.
Our documentation does not make that clear. What if we follow the same logic applied in: https://github.com/WordPress/gutenberg//blob/f6f7375d8dfa9ef52ef784e5e486c71656421770/packages/components/src/textarea-control/README.md#L98
We explicitly document:
Props not included in this set will be applied to the `a` or `button` elements.

Reply:
We should add that text as well, _but_ Button does actually have specific handling for `disabled`: if `disabled` is specified, it overrides the tag type. I can document both if you'd like.

Reply:
I went ahead and updated it for that :)

Reply:
I'm sorry @rmccue, I missed that in this case disabled was used. The changes look good to me 👍 Thank you for the improvements.